receiver/elasticapmintakereceiver/testdata/errors_expected.yaml

```diff
@@ -292,6 +196,36 @@ resourceLogs:
               - key: error.exception.code
                 value:
                   stringValue: "42"
+              - key: http.request.method
```

Comment:
These fields are the otel semconv fields that were missing (`http.*`) or moved over from the resource attributes (`url.*`)